src/services/activation/utils/index.js

```diff
@@ -0,0 +1,238 @@
+const {
+	Forbidden, NotFound, BadRequest, GeneralError,
+} = require('@feathersjs/errors');
+const { Configuration } = require('@schul-cloud/commons');
+const { HOST } = require('../../../../config/globals');
+const { equal: equalIds } = require('../../../helper/compare').ObjectId;
+const { getQuarantinedObject, createQuarantinedObject, KEYWORDS } = require('./customUtils');
+const { customErrorMessages } = require('../../helpers/utils');
+const Mail = require('../services/interface/mailFormat');
+
+/**
+ * Object of valid states of entry
+ */
+const STATE = {
+	NOT_STARTED: 'uninitiated',
+	PENDING: 'pending',
+	SUCCESS: 'success',
+	ERROR: 'error',
+};
+
+/**
+ * Deletes an Entry from db
+ * @param {*} ref				this
+ * @param {ObjectId} entryId	EntryId of Entry to delete
+ * @returns	{Object}			deleted entry
+ */
+const deleteEntry = async (ref, entryId) => {
```

Comment:
Codacy found an issue: [Expected to return a value at the end of async arrow function.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5660869)